examples/netcdf_queries/src/main.rs

```diff
@@ -0,0 +1,63 @@
+use chrono::{Utc, DateTime, Duration};
+use rust_connector_api::APIClient;
+use rust_connector_api::location::BBox;
+use rust_connector_api::errors::ConnectorError;
+
+#[tokio::main]
+async fn main(){
+    // Credentials  
+    let u_name: String = String::from("NA");
+    let u_pw: String = String::from("NA");
+    // let u_name: String = String::from("python-community");
+    // let u_pw: String = String::from("Umivipawe179");
+
+    // Create Client
+    let api: APIClient = APIClient::new(u_name,u_pw,10);
+
+    // Define the name of the file
+    let file_name = String::from("switzerland_t_2m_C.nc");
+
+    example_request(&api, &file_name).await.unwrap();
+
+    // Look at the NetCDF
+    let file = netcdf::open(&file_name).unwrap();
+    let var = &file.variable("t_2m").expect("Could not find variable 't_2m");
```

Comment:
var, file, too general names for the variables. 
Code is written to be read by other developers in the easiest way possible, just having a `file` variable, or `var` does not add to a value of the code.